apps/api/src/app/bridge/usecases/sync/sync.usecase.ts

```diff
@@ -323,6 +329,10 @@ export class Sync {
     return notificationGroupId;
   }
 
+  private getWorkflowPreferences(workflow: DiscoverWorkflowOutput): WorkflowPreferencesPartial {
+    return workflow.preferences || {};
```

Comment:
A fallback empty object is now necessary during sync after strengthening validation of the `UpsertWorkflow` use-case to disallow `undefined` preferences.